login/service_blackbox_test.go

```diff
@@ -60,6 +57,15 @@ func TestMain(m *testing.M) {
 
 	}
 
+	oauth = &oauth2.Config{
+		ClientID:     configuration.GetKeycloakClientID(),
+		ClientSecret: configuration.GetKeycloakSecret(),
+		Scopes:       []string{"user:email"},
+		Endpoint: oauth2.Endpoint{
+			AuthURL:  "http://sso.demo.almighty.io/auth/realms/fabric8/protocol/openid-connect/auth",
```

Comment:
We can remove this AuthURL and TokenURL from the Endpoint now. It will be set in the service using the default values of the dev mode. So, use Endpoint: oauth2.Endpoint{} instead.